<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tombola</class>
 <widget class="QMainWindow" name="Tombola">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1326</width>
    <height>501</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Unit√† Pastorale Cisano Bergamasco</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="playTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Gioca</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Tabellone</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="horizontalWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="column_1"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="column_2"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="column_3"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="column_4"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="column_5"/>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="column_6"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="column_7"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="column_8"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="column_9"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="column_10"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="restart">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Inizia Partita</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="extract">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Estrai Numero</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>extract</sender>
   <signal>clicked()</signal>
   <receiver>Tombola</receiver>
   <slot>extract_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>897</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>-7</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restart</sender>
   <signal>clicked()</signal>
   <receiver>Tombola</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>extract_number()</slot>
  <slot>reset()</slot>
 </slots>
</ui>
